### DISCUSSION TITLE
Issue installing/using latest version of QuSpin on Windows

### DISCUSSION THREAD

**Discussion by twricks** [QuSpin/QuSpin#689] [Q&A] [answered]
![Execution_Error_Quspin_Py310](https://github.com/user-attachments/assets/4316565e-0f4d-4d1a-96b8-291f55d53ef0)
![QuSpin_Installation_Bug](https://github.com/user-attachments/assets/1e835954-82cb-4018-aa21-b840d242d742)

Hello, I'm trying to run some Python code that uses QuSpin on a Windows machine. I was unsuccessful with Pip/manual installation as seen here [](https://quspin.github.io/QuSpin/Installation.html), when I went to install via Conda it gave this version conflict. I installed Python 3.10 and QuSpin was successfully installed, but now when I go to run the Python code it throws this Numpy version error. When I try to install Numpy 1.22 on its own, it asks for the pin-1 dependency again. What am I doing wrong?

**Comment by gcdf-ufs** [comment]
I have been experiencing the same issue on Ubuntu 22.04 and Windows 11.
![image](https://github.com/user-attachments/assets/65f7f842-3c5f-4461-9e39-9e820c4fb83e)

**Reply by twricks** [reply]
I got around this issue by installing Quspin 0.3.5 and then updating Numpy as required.

**Reply by gcdf-ufs** [reply]
Yeah. I did try to update the numpy, but it was already updated.

**Comment by jaredjeya** [comment, marked as answer]
The problem is that at some point, Numba was updated to require newer versions of Numpy than the maximum version supported on QuSpin. I'm not sure why the conflict solver isn't catching that, but a good workaround seems to be to just pin Numba to version 0.57 (potentially 0.58 will work; I haven't tested it, but certainly 0.59 is broken).
